templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,15 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+Feature levels 157-159 are reserved for future use in 6.x maintenance
+releases.
```

Comment:
I’m still not sure why we do this. We’d violate feature level monotonicity if we ever tried to make use of this reservation.

Reply:
I think we don't do this in the release commit, only after it, so we have until we next want to do a feature level bump to decide. Can you start an "api design" thread? 

I think there is a use case for this -- specifically when we're backporting a security fix that requires API changes, but we should actually reason out a workable example of that to confirm this concept makes sense, and if it doesn't, figure out some other plan for how we would handle such a scenario.